.travis.php

```diff
@@ -21,6 +21,10 @@
 
     $json = file_get_contents($dir.'/composer.json');
     $package = json_decode($json);
+    if (!$package || '{' !== $json[0]) {
+        echo "Invalid composer.json\n";
```

Comment:
Ah sorry, I meant `json_last_error_msg()` (for which we have to check the PHP version first).